utils/rpms/daos.spec

```diff
@@ -539,6 +539,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Sat Apr  2 03:56:35 UTC 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-25
```

Comment:
You need to update the `Release:` to reflect the bump to `25`.  You also need to add an entry to [debian/changelog](https://github.com/daos-stack/daos/blob/433f13fb8c6a6fa420a74324e02257f1bbd0d3e8/debian/changelog) and update the minimum mercury version for the DEB build: https://github.com/daos-stack/daos/blob/433f13fb8c6a6fa420a74324e02257f1bbd0d3e8/debian/control#L13

Reply:
The `Release:` is still at `24`:
https://github.com/daos-stack/daos/blob/7e22ccd58f940b9648d5b82fb86cf7221cd129c2/utils/rpms/daos.spec#L18

That needs to be `25`.  Unfortunately GitHub doesn't let me make code change suggestions for unmodified portions of files.